src/utils/styleUtils.js

```diff
@@ -158,10 +159,9 @@ export const highlightPointStyle = new Style({
 // Change a width according to the change of DPI (from the old geoadmin)
 // Originally introduced here https://github.com/geoadmin/mf-geoadmin3/pull/3280
 export function adjustWidth(width, dpi) {
-    if (!width) {
-        return
+    if (!width || isNaN(width)) {
```

Comment:
you should maybe also check for `dpi` in the same fashion, as if you'd let it go trough with a value of `0` you'll have a divided by zero error bellow